__tests__/get-options-overrides.spec.ts

```diff
@@ -13,6 +14,9 @@ setTypescriptModule(ts);
 const local = (x: string) => path.resolve(__dirname, x);
 const cacheDir = local("__temp/get-options-overrides");
 
+// filter expects an absolute path and resolves include/exclude to process.cwd() by default: https://github.com/ezolenko/rollup-plugin-typescript2/pull/321#discussion_r873077874
+const filtPath = (relPath: string) => normalize(`${process.cwd()}/${relPath}`);
```

Comment:
Tried both `path.normalize` (which will use the system's separators, same as `process.cwd()` I believe) and Rollup's `normalizePath` (which uses POSIX separators), and both failed...

`createFilter` uses `normalizePath` [under-the-hood](https://github.com/rollup/plugins/blob/36de7cc9c3915667bc0dd203ee7b139da8858959/packages/pluginutils/src/createFilter.ts#L16) as well.

Since only `rootDirs` and `projectReferences` failed, that got me thinking that maybe `expandIncludeWithDirs` is the issue, and indeed, it uses `path.join`, which uses the system's separators. So we get mixed separators again here, due to a bug in `expandIncludeWithDirs`: it needs to be normalized or use `posix.join` instead.

I'm gonna guess here that `rootDirs` and `projectReferences` aren't used very often (or at least not on Windows) by consumers, as this somehow hasn't come up as a bug report before

Reply:
Fixed this in #331